velox/exec/WindowPartition.h

```diff
@@ -107,9 +107,50 @@ class WindowPartition {
       vector_size_t* rawPeerStarts,
       vector_size_t* rawPeerEnds) const;
 
+  /// Sets in 'rawFrameBounds' the frame boundary for the k range
+  /// preceding/following frame.
+  /// 'isStartBound' : start or end boundary of the frame.
+  /// 'isPreceding'  : preceding or following boundary.
+  /// 'frameColumn' : column which has the range boundary for that row.
+  /// 'startRow' : starting row in the partition for this buffer computation.
+  /// 'numRows' : number of rows to compute buffer for.
+  /// 'rawPeerStarts' : buffer of peer row values for each row. If the range
```

Comment:
> If the range column is null

What does "range column" refer to? frameColumn?

Reply:
Yes, that is more precise. Thanks!